site/_apps/hn-mithril.txt

```diff
@@ -0,0 +1,26 @@
+---
+weight: 12
+title: 'Mithril HN'
+github-title: 'gyandeeps/mithril-hn'
+library: 'Mithril'
+libraries: 
+    - name: 'Mithril'
+module-bundling: 'Webpack'
+service-worker: 'Application Shell + data caching with Workbox'
+perfomance-patterns: 'HTTP/2'
+server-side-rendering: 'No'
+api: 'Node-hnapi (unoffical)'
+hosting: 'Google App Engine'
+authors: 
+    - name: 'gyandeeps'
```

Comment:
You'll need to make sure the name here matches the name in the contributor file you added so that your avatar shows underneath your app :)